raiden_contracts/tests/test_contracts_compilation.py

```diff
@@ -238,7 +222,7 @@ def test_contract_manager_constructor_does_not_invent_version() -> None:
     assert manager.contracts_version is None
 
 
-@pytest.mark.parametrize("version", [CONTRACTS_VERSION, "0.9.0", "0.3._", "0.4.0"])
+@pytest.mark.parametrize("version", [CONTRACTS_VERSION])
```

Comment:
It would be nice to have at least one more case, here. Even if you just put '0.37.0' in, it will check that we can load other versions than the current one as soon as we do the next release.